src/main/java/Main.java

```diff
@@ -0,0 +1,75 @@
+import core.basesyntax.FruitTransaction;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import service.CsvReaderService;
+import service.CsvWriterService;
+
+public class Main {
+    public static void main(String[] args) {
+        CsvReaderService csvReaderService = new CsvReaderService();
+        CsvWriterService csvWriterService = new CsvWriterService();
+
+        String inputFilePath = "input.csv";
+        String outputFilePath = "output.csv";
+
+        try {
+            List<FruitTransaction> transactions = csvReaderService.readTransactions(inputFilePath);
+
+            Map<String, Integer> fruitInventory = new HashMap<>();
```

Comment:
you violate SRP.

Make Database-class for storing purposes.

Reply:
suggest me ways to create db , and main class is ok?

Reply:
use FruitDatabase to store fruits and update its (e.g. put, remove, update)